src/themes/slider/theme.c

```diff
@@ -713,19 +709,15 @@ void add_notification_action(GtkWindow* nw, const char* text, const char* key, A
 
 	if (!gtk_widget_get_visible(windata->actions_box))
 	{
-		GtkWidget* alignment;
-
 		gtk_widget_show(windata->actions_box);
 		update_content_hbox_visibility(windata);
 
-		alignment = gtk_alignment_new(1, 0.5, 0, 0);
-		gtk_widget_show(alignment);
-		gtk_box_pack_end(GTK_BOX(windata->actions_box), alignment, FALSE, TRUE, 0);
-
 		windata->pie_countdown = gtk_drawing_area_new();
 
+		gtk_widget_set_halign (windata->actions_box, GTK_ALIGN_END);
```

Comment:
This looks weird to me... actions_box is playing role of container two lines below, and pie_countdown is packed into it... so shouldn't halign be set on pie_countdown instead? Besides, halign on actions_box is set at line 465 already.